cooperator/models/subscription_request.py

```diff
@@ -681,6 +681,11 @@ def set_membership(self):
         # To be overridden
         return True
 
+    def update_partner_info(self):
+        # To be overiden to update specific partner information based
+        # on subscription request
+        return True
```

Comment:
since it’s effectively doing nothing, i suggest this:
```suggestion
        pass
```
otherwise, it looks like this function is supposed to return a boolean value.

(and same for `set_membership()`?)

Reply:
OK, I've applied your suggestion